python/surf/misc/_McsReader.py

```diff
@@ -112,8 +117,9 @@ def open(self, filename):
                             for j in range(byteCount):
                                 # Put the address and data into a list
                                 address = baseAddr + addr + j
-                                data    = bytes[j+4]
-                                dataList.append([address, data])
+                                data    = hexBytes[j+4]
```

Comment:
any reason for the additional spaces?

Reply:
Not really.  Just easier to read the code instead on combining everything into one line.  I can combine if you think that's a better approach

Reply:
I'm not really talking about making it a one liner but instead just remove the additional whitespaces for alignment but it makes no difference other than just code formatting (https://www.python.org/dev/peps/pep-0008/). 😉  
```
data         = hexBytes[j+4]
data = hexBytes[j+4]
```

Reply:
Applied your recommendation in d35c514